### PR TITLE
Merge Packages indexes

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #87 
Added class to merge `Packages` checking for duplicates and simple test. More tests will be added in the next PR.

### CODE REVIEW

File: src/main/java/com/artipie/debian/MultiDebian.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                new BufferedReader(new InputStreamReader(gis, StandardCharsets.UTF_8));
+            String line;
+            StringBuilder item = new StringBuilder();
+            do {
+                line = rdr.readLine();
+                if (line == null || line.isEmpty()) {
+                    final String name = new ControlField.Package().value(item.toString()).get(0);
+                    final String version = new ControlField.Version().value(item.toString()).get(0);
+                    if (!(packages.containsKey(name) && packages.get(name).equals(version))) {
+                        out.write(item.append('\n').toString().getBytes(StandardCharsets.UTF_8));
```
Comment (genryxy) [inline src/main/java/com/artipie/debian/MultiDebian.java:93]: @olenagerasimova is it possible to have multiple versions for one package? In this case map will contain only the latest added version

File: src/main/java/com/artipie/debian/MultiDebian.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+ */
+public interface MultiDebian {
+
+    /**
+     * Merges provided indexes.
+     * @param items Items to merge
+     * @param res Output stream with merged data
+     * @throws IOException On IO error
+     */
+    void merge(Collection<InputStream> items, OutputStream res) throws IOException;
```
Comment (genryxy) [inline src/main/java/com/artipie/debian/MultiDebian.java:51]: @olenagerasimova we usually suppress `IOException`. Probably, do here as well

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see comments

File: src/main/java/com/artipie/debian/MultiDebian.java (hunk 2 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/debian/MultiDebian.java:51]: @genryxy hm, we usually do it on the "upper level" (working with input/output streams inside some method/class), here I work with input/output streams provided from the outside, in this case I think dealing with the exceptions is the responsibility of this method's user.

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected and answered

File: src/main/java/com/artipie/debian/MultiDebian.java (hunk 2 above)
Comment (genryxy) [reply src/main/java/com/artipie/debian/MultiDebian.java:53]: @olenagerasimova okay, got it

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**